src/ape/utils/misc.py

```diff
@@ -333,11 +334,22 @@ def inner(*args, **kwargs):
     return inner
 
 
+def get_current_timestamp() -> int:
```

Comment:
Just so it's clearer
```suggestion
def get_current_timestamp_ms() -> int:
```